doc/decisions/frontend_polygon_libraries.md

```diff
@@ -0,0 +1,46 @@
+# Polygon Libraries
+
+## Problem
+
+For drawing zones (shadings, hydrology, zones), we need to transform a set of points, drawn using a brush, into a polygon. Furthermore, we want to be able to create unions of polygons so that zones can be expanded.
+
+## Constraints
+
+- Algorithms need to be fast enough so that users won't recognize any delays.
+
+## Considered Alternatives (Hull Calculation)
+
+- graham_scan: [graham-scan](https://www.npmjs.com/package/graham_scan) is an implementation of the Graham Scan algorithm to calculate a convex hull from a given set of points.
+
+  - easy to integrate
+  - It only provides convex hull calculation.
+
+- hull.js: [hull.js](https://www.npmjs.com/package/hull.js) provides functions for calculating convex and concave hulls from a given set of points.
+  - Concavity can be configured by a parameter.
+  - Very easy to integrate and fast calculation.
+
+Note: Click [here](https://www.researchgate.net/profile/Zahrah-Yahya-Assoc-Prof-Ts-Dr/publication/312373158/figure/fig5/AS:668462426898432@1536385270015/Classification-of-convex-and-concave-hull-Adapted-from-6.png) to see an image that shows the difference between convex and concave hulls.
+
+## Considered Alternatives (Clipping Operations)
+
+- polygon-clipping: [polygon-clipping](https://www.npmjs.com/package/polygon-clipping) provides boolean polygon operations like intersection, union, and difference.
```

Comment:
If we use it it is not an alternative. The new template only has solutions and the decision is one of them.